Cql/CodeGeneration.NET/ExpressionConverter.cs

```diff
@@ -494,7 +489,7 @@ public string ConvertTopLevelFunctionDefinition(int indent, LambdaExpression fun
         // Linq.Expressions needs an explicit conversion from a value type
         // type to object, but the C# compiler will insert that boxing,
         // so we can remove those casts.
-        private static Expression StripBoxing(Expression node)
+        private static Expression stripBoxing(Expression node)
```

Comment:
weird, but there will be tons of those, as much of this code predates the marriage with our SDK. But I'll certainly rename this one!

Reply:
Weird, it's lower case in my code. What's going on here...

Reply:
oh, I see.  No, in the SDK, private methods are lower-cased.

Reply:
That's a Java-centric convention :)

Reply:
Hahaha, maybe you can still see my roots here.  I am not against changing that rule (it might well be FS is using pascal case here) - but realize that this is has been a standing convention in the SDK codebase for 10+ years, so it's going to be inconsistent for a while....  In any case, it might be nice to discuss with Kas and Marten and see what they think.

It's not just cosmetic for me - I like to make the distinction between public methods clearer, just like you can see the difference between public properties and private fields easily - I feel I need the same for methods.

Reply:
As long as we have a convention documented somewhere. This repo is currently inconsistent with following any guideline. I'm in favour of solidifying these into the editorconfig

Reply:
That's right - because it started out with Evan.  Ok, why don't we take some time this week to find out what we need to do. For now, I'll remove that .editorconfig and stick to Evan's originals.